README.md

```diff
@@ -256,12 +267,14 @@ rules:
 Visit [Basic Rule syntax](#basic-rule-syntax) for the full explanation of each
 field.
 
+## GitHub Action <a name="github-action"></a>
+
 ### Workflow configuration <a name="workflow-configuration"></a>
 
 The workflow configuration should be placed in `.github/workflows`.
 
 ```yaml
-name: PR Custom Review Status    # The PR status will be created with this name.
+name: Assign reviewers           # The PR status will be created with this name.
```

Comment:
Currently it is hardcoded as "Check reviews" at https://github.com/paritytech/pr-custom-review/blob/a18aa161dbb2f2481a9f78edd94d652d0bfbe6b4/src/core.ts#L773
To get the name from workflow configuration we can get it from `process.env.GITHUB_WORKFLOW` variable

Reply:
This change is following the pattern we use in the repos, e.g. https://github.com/paritytech/polkadot/blob/c254e5975711a6497af256f6831e9a6c752d28f5/.github/workflows/pr-custom-review.yml#L1 and suggested in https://github.com/paritytech/pr-custom-review/issues/40#issuecomment-1014577430